README.md

```diff
@@ -14,10 +14,90 @@ For an example of usage, see the [Berkeley Data 8 class](http://data8.org/).
 
 ## Installation
 
-Use `pip`:
-
+Use `pip` to install the package: 
 ```
 pip install datascience
+
+```
+
+To verify that the package is installed correctly, run:
+```
+python -c "import datascience; print(datascience.__version__)"
+
+```
+
+## Quick Start Guide 
```

Comment:
I think this idea of putting a Getting Started experience on the README - but I think we're better off linking to this page, which is much more visual and comprehensive: https://www.data8.org/datascience/tutorial.html#getting-started